src/main/java/org/opensearch/ad/task/ADBatchTaskRunner.java

```diff
@@ -127,7 +125,8 @@ public class ADBatchTaskRunner {
     private final FeatureManager featureManager;
     private final ADCircuitBreakerService adCircuitBreakerService;
     private final ADTaskManager adTaskManager;
-    private final AnomalyResultBulkIndexHandler anomalyResultBulkIndexHandler;
+    // @anomaly-detection - commented until we have support for SDKRestClient.prepareBulk()
```

Comment:
Do we an issue to track this?

Reply:
Seems that we do not, I went ahead and created one here : https://github.com/opensearch-project/opensearch-sdk-java/issues/635. Ill add these to the comments